### PR TITLE
en, zh: add release notes for v1.2.0-beta.1

### PR DESCRIPTION
**Pull request by handlerww**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Add release notes for TiDB-Operator v1.2.0-alpha.1

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
    - pingcap/tidb-operator#3897

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/releases/release-1.2.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: TiDB Operator 1.2.0-beta.1 Release Notes
+---
+
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
```
Comment (ran-huang) [inline en/releases/release-1.2.0-beta.1.md:6]:
```suggestion

Release date: April 7, 2021

TiDB Operator version: 1.2.0-beta.1

```

File: en/releases/release-1.2.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Support setting customized envs for backup and restore jobs ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
+- TidbMonitor add additional volume and volumeMount configurations. ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
+- Support affinity and tolerations in backup/restore jobs. ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
+
+## Improvements
+
+- Fix PVC resize for TiDB. ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
+- Fix support for multiple PVC for PD. ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
+- Fix support for multiple PVC for TiKV. ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
+- Fix the issue that multiple PVC is treated wrong in PVC resizer. ([#3858](https://github.com/pingcap/tidb-operator/pull/3858), [@dragonly](https://github.com/dragonly))
```
Comment (ran-huang) [inline en/releases/release-1.2.0-beta.1.md:18]: This should be in `bug fixes` category.

Comment (handlerww) [reply en/releases/release-1.2.0-beta.1.md:29]: done

File: en/releases/release-1.2.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
+Release date: April 7, 2021
+
+TiDB Operator version: 1.2.0-beta.1
+
+## New Features
+
+- Support setting customized envs for backup and restore jobs ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
+- TidbMonitor add additional volume and volumeMount configurations. ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:14]:
```suggestion
- TidbMonitor add additional volume and volumeMount configurations ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- Support affinity and tolerations in backup/restore jobs. ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:15]:
```suggestion
- Support affinity and tolerations in backup/restore CR ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## Improvements
+
+- Add retry for DNS lookup failure exception in TiDBInitializer. ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:19]:
```suggestion
- Add retry for DNS lookup failure exception in TiDBInitializer ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- The resources in the tidb-operator chart use the new service account when `appendReleaseSuffix` is set to `true` ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Optimze thanos example yaml files. ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:21]:
```suggestion
- Optimze thanos example yaml files ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- Delete evict leader scheduler after TiKV Pod is recreated during upgrade ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
+- Support multiple PVC for PD. ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:23]:
```suggestion
- Support multiple PVC for PD ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+- Support multiple PVC for TiKV. ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:24]:
```suggestion
- Support multiple PVC for TiKV ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+## Bug Fixes
+
+- Fix PVC resize for TiDB. ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:28]:
```suggestion
- Support PVC resizing for TiDB ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```

Comment (DanielZhangQD) [reply en/releases/release-1.2.0-beta.1.md:28]: Move this to the `Improvements` section?

File: en/releases/release-1.2.0-beta.1.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+- Fix the issue that multiple PVC is treated wrong in PVC resizer. ([#3858](https://github.com/pingcap/tidb-operator/pull/3858), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:29]:
```suggestion
- Fix the issue that PVCs will be set to incorrect size if multiple PVCs are configured for PD/TiKV ([#3858](https://github.com/pingcap/tidb-operator/pull/3858), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+- Fix the issue when TidbCluster's `.spec.tidb` is unset. ([#3852](https://github.com/pingcap/tidb-operator/pull/3852), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:30]:
```suggestion
- Fix the panic issue when `.spec.tidb` is not set in the TidbCluster CR with TLS enabled ([#3852](https://github.com/pingcap/tidb-operator/pull/3852), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+- Fix `UnjoinedMembers` in PD and DM cluster status. ([#3836](https://github.com/pingcap/tidb-operator/pull/3836), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:31]: Remove this.

File: en/releases/release-1.2.0-beta.1.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+- Fix tidbmonitor externallabels unrecognized environment variables bug. ([#3785](https://github.com/pingcap/tidb-operator/pull/3785), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:32]:
```suggestion
- Fix the issue that some unrecognized environment variables are included in the external labels of the TidbMonitor ([#3785](https://github.com/pingcap/tidb-operator/pull/3785), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.2.0-beta.1.md (hunk 6 above)
Comment (DanielZhangQD) [reply en/releases/release-1.2.0-beta.1.md:21]: I think we can remove this.

**Comment by ran-huang** [comment]
Please add Chinese release notes.

File: en/releases/release-1.2.0-beta.1.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
+Release date: April 7, 2021
+
+TiDB Operator version: 1.2.0-beta.1
+
+## New Features
+
+- Support setting customized envs for backup and restore jobs ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
```
Comment (dragonly) [inline en/releases/release-1.2.0-beta.1.md:13]:
```suggestion
- Support setting customized envs for backup and restore job containers ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+- Support affinity and tolerations in backup/restore CR ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
+
+## Improvements
+
+- Add retry for DNS lookup failure exception in TiDBInitializer ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
+- The resources in the tidb-operator chart use the new service account when `appendReleaseSuffix` is set to `true` ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Optimze thanos example yaml files ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
+- Delete evict leader scheduler after TiKV Pod is recreated during upgrade ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
+- Support multiple PVC for PD ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
+- Support multiple PVC for TiKV ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```
Comment (dragonly) [inline en/releases/release-1.2.0-beta.1.md:24]:
```suggestion
- Improve support multiple PVC for PD ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
- Improve support multiple PVC for TiKV ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```

**Review by dragonly — COMMENTED** [review]
rest LGTM

File: en/releases/release-1.2.0-beta.1.md (hunk 14 above)
Comment (ran-huang) [reply en/releases/release-1.2.0-beta.1.md:13]: `envs` -> `environment variables`?

Comment (handlerww) [reply en/releases/release-1.2.0-beta.1.md]: fixed

File: en/releases/release-1.2.0-beta.1.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+- Support affinity and tolerations in backup/restore CR ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
+
+## Improvements
+
+- Add retry for DNS lookup failure exception in TiDBInitializer ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
+- The resources in the tidb-operator chart use the new service account when `appendReleaseSuffix` is set to `true` ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Optimze thanos example yaml files ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
+- Delete evict leader scheduler after TiKV Pod is recreated during upgrade ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
+- Improve support multiple PVC for PD ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:23]:
```suggestion
- Support multiple PVCs for PD during scaling and failover ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+- Improve support multiple PVC for TiKV ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:24]:
```suggestion
- Support multiple PVCs for TiKV during scaling ([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+- Improve support PVC resizing for TiDB ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline en/releases/release-1.2.0-beta.1.md:25]:
```suggestion
- Support PVC resizing for TiDB ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
+发布日期: April 7, 2021
+
+TiDB Operator 版本: 1.2.0-beta.1
+
+## 新功能
+
+- 支持设置备份和恢复的 Job 容器的自定义环境变量 ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:13]: 标点符号都是英文的

Comment (DanielZhangQD) [reply zh/releases/release-1.2.0-beta.1.md:13]:
```suggestion
- 支持为备份和恢复 Job 设置自定义环境变量 ([#3833](https://github.com/pingcap/tidb-operator/pull/3833)，[@dragonly](https://github.com/dragonly))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+- TidbMonitor 支持添加 volume 和 volumeMount 配置 ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:14]:
```suggestion
- 支持为 TidbMonitor 配置额外的 volume 和 volumeMount ([#3855](https://github.com/pingcap/tidb-operator/pull/3855)，[@mikechengwei](https://github.com/mikechengwei))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+- 支持备份恢复 CR 设置 affinity 和 tolerations ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
+- 设置 `appendReleaseSuffix` 为 `true` 时, 支持 tidb-operator chart 使用新的 service account ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+
+## 优化提升
+
+- TiDBInitializer 中增加重试机制, 解决 DNS 查询异常处理问题 ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
+- 优化 Thanos 的 example yaml ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
+- 滚动更新过程中, 调整 EndEvictLeader 时机到 TiKV Pod 重建之后 ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:22]:
```suggestion
- 滚动更新过程中, 等待 TiKV Pod 升级完成之后再删除 evict leader scheduler  ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+- 在 PD 的扩缩容和容灾过程中增加多 PVC 支持 ([#3820](https://github.com/pingcap/tidb-operator/pull/3820), [@dragonly](https://github.com/dragonly))
+- 在 TiKV 的扩缩容过程中增加多 PVC 支持([#3816](https://github.com/pingcap/tidb-operator/pull/3816), [@dragonly](https://github.com/dragonly))
+- 支持 TiDB 调整 PVC 容量 ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:25]:
```suggestion
- 支持调整 TiDB PVC 容量 ([#3891](https://github.com/pingcap/tidb-operator/pull/3891), [@dragonly](https://github.com/dragonly))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 23]
```diff
@@ -0,0 +1,31 @@
...
+
+## Bug 修复
+
+- 修复 PD/TiKV 挂载多 PVC 时容量设置错误的问题 ([#3858](https://github.com/pingcap/tidb-operator/pull/3858), [@dragonly](https://github.com/dragonly))
+- 修复开启 TLS 后, TidbCluster 的 `.spec.tidb` 为空时 tidb-controller-manager panic 的问题 ([#3852](https://github.com/pingcap/tidb-operator/pull/3852), [@dragonly](https://github.com/dragonly))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:30]:
```suggestion
- 修复创建 `.spec.tidb` 为空并开启 TLS 的 TidbCluster 导致 tidb-controller-manager panic 的问题 ([#3852](https://github.com/pingcap/tidb-operator/pull/3852), [@dragonly](https://github.com/dragonly))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 24]
```diff
@@ -0,0 +1,31 @@
...
+- 修复一些未识别的环境变量被包含在 TidbMonitor 外部标签的问题 ([#3785](https://github.com/pingcap/tidb-operator/pull/3785), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:31]:
```suggestion
- 修复 TidbMonitor 外部标签包含一些无法识别的环境变量的问题 ([#3785](https://github.com/pingcap/tidb-operator/pull/3785), [@mikechengwei](https://github.com/mikechengwei))
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/releases/release-1.2.0-beta.1.md [hunk 25]
```diff
@@ -0,0 +1,31 @@
+---
+title: TiDB Operator 1.2.0-beta.1 Release Notes
+---
+
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
+发布日期: April 7，2021
```
Comment (ran-huang) [inline zh/releases/release-1.2.0-beta.1.md:7]:
```suggestion
发布日期: 2021 年 4 月 7 日
```

File: en/releases/release-1.2.0-beta.1.md [hunk 26]
```diff
@@ -0,0 +1,31 @@
...
+
+- Support setting customized environment variables for backup and restore job containers ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
+- TidbMonitor add additional volume and volumeMount configurations ([#3855](https://github.com/pingcap/tidb-operator/pull/3855), [@mikechengwei](https://github.com/mikechengwei))
+- Support affinity and tolerations in backup/restore CR ([#3835](https://github.com/pingcap/tidb-operator/pull/3835), [@dragonly](https://github.com/dragonly))
+- The resources in the tidb-operator chart use the new service account when `appendReleaseSuffix` is set to `true` ([#3819](https://github.com/pingcap/tidb-operator/pull/3819), [@DanielZhangQD](https://github.com/DanielZhangQD))
+
+## Improvements
+
+- Add retry for DNS lookup failure exception in TiDBInitializer ([#3884](https://github.com/pingcap/tidb-operator/pull/3884), [@handlerww](https://github.com/handlerww))
+- Optimze thanos example yaml files ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (ran-huang) [inline en/releases/release-1.2.0-beta.1.md:21]:
```suggestion
- Optimize thanos example yaml files ([#3726](https://github.com/pingcap/tidb-operator/pull/3726), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.2.0-beta.1.md [hunk 27]
```diff
@@ -0,0 +1,31 @@
...
+- EndEvictLeader after TiKV Pod is recreated in RollingUpdate. ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```
Comment (ran-huang) [inline en/releases/release-1.2.0-beta.1.md:22]:
```suggestion
- Delete the evict leader scheduler after TiKV Pod is recreated during the rolling update ([#3724](https://github.com/pingcap/tidb-operator/pull/3724), [@handlerww](https://github.com/handlerww))
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 28]
```diff
@@ -0,0 +1,31 @@
...
+
+TiDB Operator 版本: 1.2.0-beta.1
```
Comment (ran-huang) [inline zh/releases/release-1.2.0-beta.1.md:9]:
```suggestion
TiDB Operator 版本：1.2.0-beta.1
```

File: zh/releases/release-1.2.0-beta.1.md [hunk 29]
```diff
@@ -0,0 +1,31 @@
+---
+title: TiDB Operator 1.2.0-beta.1 Release Notes
+---
+
+# TiDB Operator 1.2.0-beta.1 Release Notes
+
+发布日期: 2021 年 4 月 7 日
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.0-beta.1.md:7]:
```suggestion
发布日期：2021 年 4 月 7 日
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**